apps/rod/behaviours/send-email-notification.js

```diff
@@ -0,0 +1,274 @@
+  }
+
+  if (whoCompleted === 'sponsor') {
+    return getLabelForField(req, 'sponsor-type');
+  }
+
+  if (whoCompleted === 'guardian') {
+    return getLabelForField(req, 'dependant-or-guardian');
+  }
+
+  return '';
+};
+
+const getApplicationCategory = req => {
+  const applicationType = req.sessionModel.get('application-type');
+
+  if (applicationType === 'visa') {
+    return getLabelForField(req, 'visa-type');
+  }
+
+  if (applicationType === 'further-leave') {
+    return getLabelForField(req, 'further-leave-to-remain');
+  }
+
+  return getLabelForField(req, 'application-type');
+};
+
+const getUserDetails = req => {
+  return {
+    record_number: req.sessionModel.get('uniqueRodReference'),
+    person_completing: getWhoCompletedForm(req),
+    application_category: getApplicationCategory(req),
+    application_submitted_date: dateFormatter.format(
+      new Date(getValueOfDefault(req, 'date-of-application'))
+    ),
+    has_cancel_application: getYesOrNoStr(req, 'cancel-application'),
+    cancel_application: getLabelForField(req, 'cancel-application'),
+    has_applicant_passport: getYesOrNoStr(
+      req,
+      'is-requesting-passport-to-travel'
+    ),
+    applicant_passport: getLabelForField(
+      req,
+      'is-requesting-passport-to-travel'
+    ),
+    reference_number: getValueOfDefault(req, 'selectedRefNumbers'),
+    has_case_id: getYesOrNoStr(req, 'rod-case-id'),
+    case_id: getValueOfDefault(req, 'rod-case-id'),
+    has_ho_reference_number: getYesOrNoStr(req, 'rod-ho-reference-number'),
+    ho_reference_number: getValueOfDefault(req, 'rod-ho-reference-number'),
+    has_payment_reference_number: getYesOrNoStr(
+      req,
+      'rod-payment-reference-number'
+    ),
+    payment_reference_number: getValueOfDefault(
+      req,
+      'rod-payment-reference-number'
+    ),
+    has_courier_reference_number: getYesOrNoStr(
+      req,
+      'rod-courier-reference-number'
+    ),
+    courier_reference_number: getValueOfDefault(
+      req,
+      'rod-courier-reference-number'
+    ),
+    has_your_documents: getYesOrNoStr(req, 'yourDocuments'),
+    your_documents: getValueOfDefault(req, 'yourDocuments'),
+    has_document_description: getYesOrNoStr(req, 'document-description'),
+    document_description: getValueOfDefault(req, 'document-description'),
+    applicant_full_name: getValueOfDefault(req, 'main-applicant-full-name'),
+    applicant_dob: dateFormatter.format(
+      new Date(getValueOfDefault(req, 'main-applicant-dob'))
+    ),
+    applicant_nationality: getValueOfDefault(req, 'main-applicant-nationality'),
+    applicant_address: getValueOfDefault(req, 'applicantAddress'),
+    delivery_address: getValueOfDefault(req, 'deliveryAddress'),
+    contact_email: getValueOfDefault(req, 'contact-email'),
+    contact_telephone: getValueOfDefault(req, 'contact-telephone'),
+    has_notes: getYesOrNoStr(req, 'notes'),
+    notes: getValueOfDefault(req, 'notes')
+  };
+};
+
+const visaTypeEmailMap = [
```

Comment:
This is a conscious decision. Appears to be redundant as of now however, this will allow us to map one to one email box in the future if required.